driver.sh

```diff
@@ -129,6 +129,14 @@ build_linux() {
   # is good about figuring out what needs to be rebuilt
   [[ -n "${cleanup:-}" ]] && mako_reactor mrproper
   mako_reactor ${config}
+  # If we're using a defconfig, enable some more common config options
+  # like debugging, selftests, and common drivers
+  if [[ ${config} =~ defconfig ]]; then
```

Comment:
was `=~` intentional (regex) vs just `==` for string equality?

Reply:
Yes because of `multi_v7_defconfig` (although that's the target of `ARCH=arm make defconfig` anyways so maybe that should be updated)